src/main/java/trello/api/rest/RequestManager.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * @(#) RunCukesTest.java Copyright (c) 2019 Jala Foundation.
```

Comment:
Please remove the name of class. "@(#) RunCukesTest.java"

Reply:
Solved